ACPs/77-reinventing-subnets/README.md

```diff
@@ -1,175 +1,180 @@
+
+Though Subnets have always managed their own consensus mechanisms, transaction processing, and security protocols directly within their own networks, the added functionality introduced in this ACP will solidify their status as standalone networks, or layer 1 blockchains.
+
 ## Specification
 
-At a high-level, Subnets can manage their validator sets externally to the P-Chain by setting the blockchain ID and address of their _validator manager_. The P-Chain will consume Warp messages that modify the Subnet's validator set. To confirm modification of the Subnet's validator set, the P-Chain will also produce Warp messages. Subnet Validators will no longer be required to validate the Primary Network, removing the 2000 $AVAX stake requirement to be a Subnet Validator. In place of the stake requirement, a continuous fee denominated in $AVAX is introduced to maintain an active Subnet Validator. Subnet Validators are only required to sync the P-Chain (not X/C-Chain) in order to track validator set changes and support cross-Subnet communication.
+At a high-level, L1s can manage their validator sets externally to the P-Chain by setting the blockchain ID and address of their _validator manager_. The P-Chain will consume Warp messages that modify the L1's validator set. To confirm modification of the L1's validator set, the P-Chain will also produce Warp messages. L1 validators will no longer be required to validate the Primary Network, removing the 2000 $AVAX stake requirement previously imposed on Subnet validators. In place of the stake requirement, a continuous fee denominated in $AVAX is introduced to maintain an active L1 validator. L1 validators are only required to sync the P-Chain (not X/C-Chain) in order to track validator set changes and support cross-L1 communication.
 
 ### P-Chain Warp Message Payloads
 
-To enable management of a Subnet's validator set externally to the P-Chain, Warp message verification will be added to the [`PlatformVM`](https://github.com/ava-labs/avalanchego/tree/master/vms/platformvm). For a Warp message to be considered valid by the P-Chain, at least 67% of the `sourceChainID`'s weight must have participated in the aggregate BLS signature. This is equivalent to the threshold set for the C-Chain. A future ACP may be proposed to support modification of this threshold on a per-Subnet basis.
+To enable management of an L1's validator set externally to the P-Chain, Warp message verification will be added to the [`PlatformVM`](https://github.com/ava-labs/avalanchego/tree/master/vms/platformvm). For a Warp message to be considered valid by the P-Chain, at least 67% of the `sourceChainID`'s weight must have participated in the aggregate BLS signature. This is equivalent to the threshold set for the C-Chain. A future ACP may be proposed to support modification of this threshold on a per-L1 basis.
```

Comment:
```suggestion
To enable management of a L1's validator set externally to the P-Chain, Warp message verification will be added to the [`PlatformVM`](https://github.com/ava-labs/avalanchego/tree/master/vms/platformvm). For a Warp message to be considered valid by the P-Chain, at least 67% of the `sourceChainID`'s weight must have participated in the aggregate BLS signature. This is equivalent to the threshold set for the C-Chain. A future ACP may be proposed to support modification of this threshold on a per-L1 basis.
```

Reply:
> these indefinite articles are used [according to the sound](https://www.merriam-webster.com/grammar/is-it-a-or-an) that starts the following word, not the letter. So, we say “a dog” and “a balloon” or “an urgent message” and “an ant” but also “a useless pen” and “an hour.”

"L" said _out loud_ begins with a vowel sound; therefore the appropriate article is "an".
https://www.pristineword.com/a-an/#tit05

Reply:
TIL